test.js

```diff
@@ -1,8 +1,7 @@
-'use strict';
-var test = require('ava');
-var camelcaseKeys = require('./');
+import test from 'ava';
+import camelcaseKeys from './';
 
-test(function (t) {
-	t.assert(camelcaseKeys({'foo-bar': true}).fooBar);
+test('camelcase keys', t => {
```

Comment:
`✓ [anonimous]` is kind of strange, would be cool if `AVA` knew what's up and called the only test `default` for example. What do you think?